internal/utils/queue.go

```
@@ -0,0 +1,175 @@
+			case <-timeoutChan:
+				return
+
+			case d, ok := <-msgs:
+				if !ok {
+					return
+				}
+
+				var event EventData
+				err := json.Unmarshal(d.Body, &event)
+				if err != nil {
+					d.Nack(false, false)
```

golangci-lint (1.64.8)

Error return value of `d.Nack` is not checked

(errcheck)